website_sale_default_country/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+eCommerce Default Country
+=========================
+
+This module extends the functionality of eCommerce to provide a sane default
```

Comment:
This module extends the functionality of eCommerce to default the country value during 
your customers' checkout process country and allow them to avoid digging in
the endless countries list.